frontned/src/styles.ts

```diff
@@ -0,0 +1,7 @@
+import { createTheme } from '@mui/material/styles';
```

Comment:
I would recommend to change the name type to theme.ts cause this is actually what u export from this file

Reply:
And move it to some seperate folder ;)